ets-ecole-de-technologie-superieure.csl

```diff
@@ -122,21 +122,28 @@
     <text variable="issue" prefix=" "/>
   </macro>
   <macro name="access">
-    <group delimiter=". ">
-      <text variable="URL" prefix="&lt;" suffix="&gt;"/>
-      <choose>
-        <if variable="URL" match="any">
-          <group delimiter=" ">
-            <text value="Consulté le"/>
-            <date variable="accessed">
-              <date-part name="day" suffix=" "/>
-              <date-part name="month" suffix=" "/>
-              <date-part name="year"/>
-            </date>
-          </group>
-        </if>
-      </choose>
-    </group>
+		<choose>
```

Comment:
Sorry about the indenting. I always seem to have a problem with that when I copy my modifs. into GitHub. It might be my text editor giving me troubles... I will check into it for next time.